### DISCUSSION TITLE
flink 对接 iotdb 问题咨询

### DISCUSSION THREAD

**Discussion by zjf0128** [apache/iotdb#11195] [Q&A]
关于flink-sql的支持，后续是不是用sql的模式就可以了，还是说还是得写代码用实现。
比如可以：
CREATE TABLE ads_project_pro_project(
  id bigint,
  `project_cost` DECIMAL(38, 0),
  `number` bigint,
  `handle_time` bigint,
  PRIMARY KEY (`id`) NOT ENFORCED
)
 WITH (
'connector' = 'jdbc',
'url' = 'jdbc:iotdb://192.168.1.1:6667/data_lake_warehouse?serverTimezone=Asia/Shanghai&useSSL=false',
'username' = 'lj_dev',
'password' = '1qaz@WSX#EDC');  类似这样的实现呢。

**Comment by wanghui42** [comment]
是类似这样的实现，用 SQL。不过列名和参数不是这样
